src/Microsoft.AspNetCore.Mvc.RazorPages/ApplicationFeature/CompiledPageInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Mvc.RazorPages.ApplicationFeature
```

Comment:
Guess I still need this. Presumably we can change this to the assembly attribute once we change how we do the compilation.